angular-workspace/projects/ni/spright-angular/internal-utilities/template-value-helpers.ts

```diff
@@ -0,0 +1,61 @@
+/**
```

Comment:
Thoughts on ways to avoid duplication of these helpers? How about a `nimble-angular-utilities` package that both `nimble-angular` and `spright-angular` depend on?

Reply:
I think internal `@ni/nimble-angular/internal-utilities` is already a publicly accessible entry-point. We can start by depending on that since it's already there

Reply:
We're not actually using any of those helpers in Rectangle, so it will just be up to future devs to use them from `@ni/nimble-angular/internal-utilities`.